tests/scripts/thread-cert/backbone/test_mlr_multicast_routing_across_thread_pans.py

```diff
@@ -0,0 +1,230 @@
+        self.assertEqual('router', self.nodes[ROUTER1].get_state())
+        self.nodes[ROUTER2].add_ipmaddr(MA1)
+        self.nodes[ROUTER2].add_ipmaddr(MA2)
+        self.simulator.go(WAIT_REDUNDANCE)
+
+        self.collect_ipaddrs()
+
+        # ping MA1 from Host could get replied from R1 and R2
+        self.assertTrue(self.nodes[HOST].ping(MA1, backbone=True, ttl=5))
+        self.simulator.go(WAIT_REDUNDANCE)
+
+        # ping MA2 from R1 could get replied from Host and R2
+        # TODO (DUA): ping reply can not reach ROUTER1 since DUA feature is not complete.
+        self.assertFalse(self.nodes[ROUTER1].ping(MA2))
+        self.simulator.go(WAIT_REDUNDANCE)
+
+    def verify(self, pv: PacketVerifier):
+        pkts = pv.pkts
+        pv.add_common_vars()
+        pv.summary.show()
+
+        PBBR1 = pv.vars['PBBR1']
+        PBBR1_ETH = pv.vars['PBBR1_ETH']
+        PBBR2 = pv.vars['PBBR2']
+        PBBR2_ETH = pv.vars['PBBR2_ETH']
+        ROUTER1 = pv.vars['ROUTER1']
+        ROUTER2 = pv.vars['ROUTER2']
+        HOST_ETH = pv.vars['Host_ETH']
+        HOST_BGUA = pv.vars['Host_BGUA']
+
+        ROUTER1_DUA = pv.vars['ROUTER1_DUA']
+        ROUTER2_DUA = pv.vars['ROUTER2_DUA']
+
+        #
+        # Verify Host ping MA1 to R1 and R2
+        #
+
+        # Host should send ping MA1 in the Backbone link
+        ping_ma1 = pkts.filter_eth_src(HOST_ETH).filter_ipv6_src_dst(HOST_BGUA, MA1).filter_ping_request().must_next()
+
+        with pkts.save_index():
+            # PBBR1 should forward ping-MA1 to Router1
+            pkts.filter_wpan_src64(PBBR1).filter_AMPLFMA().filter_ping_request(
+                identifier=ping_ma1.icmpv6.echo.identifier).must_next()
+            # Router1 should send ping reply
+            ping_reply_pkts = pkts.filter_ipv6_src_dst(
+                ROUTER1_DUA, HOST_BGUA).filter_ping_reply(identifier=ping_ma1.icmpv6.echo.identifier)
+            ping_reply_pkts.filter_wpan_src64(ROUTER1).must_next()
+            # PBBR1 should forward ping reply to the Backbone link
+            ping_reply_pkts.filter_eth_src(PBBR1_ETH).must_next()
+
+        # PBBR2 should forward ping-MA1 to Router2
+        pkts.filter_wpan_src64(PBBR2).filter_AMPLFMA().filter_ping_request(
+            identifier=ping_ma1.icmpv6.echo.identifier).must_next()
+        # Router2 should send ping reply
+        ping_reply_pkts = pkts.filter_ipv6_src_dst(
+            ROUTER2_DUA, HOST_BGUA).filter_ping_reply(identifier=ping_ma1.icmpv6.echo.identifier)
+        ping_reply_pkts.filter_wpan_src64(ROUTER2).must_next()
+        # PBBR2 should forward ping reply to the Backbone link
+        ping_reply_pkts.filter_eth_src(PBBR2_ETH).must_next()
+
+        #
+        # Verify R1 pings MA2 to Host and R2
+        #
+
+        # ROUTER1 should send the multicast ping request
+        ping_ma2 = pkts.filter_wpan_src64(ROUTER1).filter_AMPLFMA().filter_ping_request().filter(
+            lambda p: p.ipv6.hlim == 64).must_next()
```

Comment:
One suggest here is may use seed id to find the packet that originated from ROUTER1

Reply:
revised to check MPL seed_id